docs/source/api/v3/deliveryservice_requests_id_assign.rst

```diff
@@ -13,255 +13,287 @@
-	:consistentHashRegex:           A :ref:`ds-consistent-hashing-regex`
-	:deepCachingType:               The :ref:`ds-deep-caching` setting for this :term:`Delivery Service`
-	:displayName:                   The :ref:`ds-display-name`
-	:dnsBypassCname:                A :ref:`ds-dns-bypass-cname`
-	:dnsBypassIp:                   A :ref:`ds-dns-bypass-ip`
-	:dnsBypassIp6:                  A :ref:`ds-dns-bypass-ipv6`
-	:dnsBypassTtl:                  The :ref:`ds-dns-bypass-ttl`
-	:dscp:                          A :ref:`ds-dscp` to be used within the :term:`Delivery Service`
-	:ecsEnabled:                    A boolean that defines the :ref:`ds-ecs` setting on this :term:`Delivery Service`
-	:edgeHeaderRewrite:             A set of :ref:`ds-edge-header-rw-rules`
-	:exampleURLs:                   An array of :ref:`ds-example-urls`
-	:firstHeaderRewrite:            A set of :ref:`ds-first-header-rw-rules`
-	:fqPacingRate:                  The :ref:`ds-fqpr`
-	:geoLimit:                      An integer that defines the :ref:`ds-geo-limit`
-	:geoLimitCountries:             A string containing a comma-separated list defining the :ref:`ds-geo-limit-countries`\ [#geolimit]_
-	:geoLimitRedirectUrl:           A :ref:`ds-geo-limit-redirect-url`\ [#geolimit]_
-	:geoProvider:                   The :ref:`ds-geo-provider`
-	:globalMaxMbps:                 The :ref:`ds-global-max-mbps`
-	:globalMaxTps:                  The :ref:`ds-global-max-tps`
-	:httpBypassFqdn:                A :ref:`ds-http-bypass-fqdn`
-	:id:                            An integral, unique identifier for this :term:`Delivery Service`
-	:infoUrl:                       An :ref:`ds-info-url`
-	:initialDispersion:             The :ref:`ds-initial-dispersion`
-	:innerHeaderRewrite:            A set of :ref:`ds-inner-header-rw-rules`
-	:ipv6RoutingEnabled:            A boolean that defines the :ref:`ds-ipv6-routing` setting on this :term:`Delivery Service`
-	:lastHeaderRewrite:             A set of :ref:`ds-last-header-rw-rules`
-	:lastUpdated:                   The date and time at which this :term:`Delivery Service` was last updated, in :rfc:`3339` format
-	:logsEnabled:                   A boolean that defines the :ref:`ds-logs-enabled` setting on this :term:`Delivery Service`
-	:longDesc:                      The :ref:`ds-longdesc` of this :term:`Delivery Service`
-	:longDesc1:                     An optional field containing the :ref:`ds-longdesc2` of this :term:`Delivery Service`
-	:longDesc2:                     An optional field containing the :ref:`ds-longdesc3` of this :term:`Delivery Service`
-	:matchList:                     The :term:`Delivery Service`'s :ref:`ds-matchlist`
+It is not required to send both of these; either property is sufficient to determine an :ref:`dsr-assignee`. In most cases, it's easier to use just `assignee`. If both *are* given, then `assigneeId` will take precedence in the event that the two properties do not refer to the same user.
```

Comment:
This should also mention that providing neither property unassigns a DSR from a user.